JSON/req-schema.json

```diff
@@ -0,0 +1,140 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/perftool-incubator/multiplex/master/req-schema.json",
+
+    "type": "object",
+    "properties": {
+        "presets": {
+            "type": "object",
+            "patternProperties": {
+                "^[a-zA-Z0-9_-]*$": {
+                    "type": "array",
+                    "minItems": 1,
+                    "uniqueItems": true,
+                    "items": {
+                        "$ref": "#/definitions/mv_param"
+                    }
+                }
+            },
+            "additionalProperties": false
+        },
+        "validations": {
+            "type": "object",
+            "patternProperties": {
+                "^[A-Za-z]+[A-Za-z0-9_-]*$": {
```

Comment:
Just curious...are you saying that for the validations the property name has to start with a letter?  If so, why?

Reply:
Is there any param called "99frame-size" or "1st-param-name" ? With this pattern you can have numbers, but not at the beginning. I can change if this is a requirement. Also, all the regex patterns in the json files can be easily changed if you need (even locally without a PR).

Reply:
I think we can't predict what the parameter names are so we should try to be as flexible as possible.

Reply:
ack